en/pocket_privacy_policy.md

```diff
@@ -0,0 +1,127 @@
+
+## USE OF COOKIES AND ANALYTICS
+
+### Types and Purposes
+
+We use cookies and analytics tools for the following purposes:
+
+* Necessary: These cookies are necessary for our website to function and to support our business. They cannot be switched off. For example, we use them to manage your login state, and our ad partners use them to display ads and measure impressions and clicks. You can set your browser to block or alert you about these cookies, but this may cause parts of the site to not work properly. These cookies do not store any personally identifiable information and unless Personalized Advertising Cookies are enabled, ads are non-personalized.
+* Analytics: We place these cookies on your device to help us understand how users engage with our website and product, and to remember your account preferences and language settings. For example, we use Google Analytics and other tools to understand which Pocket webpages are popular. We also use pixels in our emails in order to collect usage information.
+* Personalized Advertising: Some Pocket web pages have ads. If you choose, Pocket ad partners will place advertising cookies on your device to personalize the ads you see here and on other websites. If you play a YouTube video on Pocket, YouTube may place cookies on your device which can be used for personalization.
+
+### Your Choices
+
+* Cookie Preference tool: By default, Pocket sets necessary and analytics cookies, and disables personalized advertising cookies. You can manage these cookies from this Cookie Preferences tool. [Learn more](https://help.getpocket.com/article/1163-pocket-website-cookies-faq).
+* Your browser: Your browser preferences menu can help you manage cookies and prevent cross-site tracking.
+* Google’s Analytics opt-out: Install the [Google Analytics Opt-out Browser Add-on](https://tools.google.com/dlpage/gaoptout), which prohibits data transmission to Google Analytics.
+* External opt-out tools: Some third- party organizations provide ways of opting out of personalized advertising delivered by participating companies. For information about how to opt-out, you can visit these organizations’ websites: [Network Advertising Initiative’s opt-out tools](https://optout.networkadvertising.org/?c=1), the [Digital Advertising Initiative’s Browser Check tool](https://optout.aboutads.info/?c=2&lang=EN), and the [European Interactive Digital Advertising Alliance Your Ad Choices](https://www.youronlinechoices.com/uk/your-ad-choices).
```

Comment:
The second link should probably go without `lang=EN`, it's unfortunate that the third forces you to select a country.

Reply:
I thought about it, but that website is confusing. From the home page, I had no idea where to click next, so it's probably safer to keep the link as is.

Reply:
Cool - reverted the change to the third link